src/legacy/core_plugins/kibana_react/public/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// TODO these are imports from the old plugin world.
+// Once the new platform is ready, they can get removed
+// and handled by the platform itself in the setup method
+// of the ExpressionExectorService
```

Comment:
Naive question: Would it still work to put this stuff in `src/plugins/kibana_react` rather than creating a duplicate legacy plugin?

The only two places I see it importing from the legacy world are `SearchBar` in the `data` plugin, and some scss files. So can you still statically import from those places in the new platform? Or are linting rules preventing this?

Reply:
a. We don't have a way to load css files from the new platform plugin yet.
b. In a following PR I also moved SearchBar into `kibana_react`

Anyway, I prefer to move it to new platform in a separate PR, after I merge the follow up PRs.
Hope it's cool :-P

Reply:
Ah yeah forgot about the css thing. 👍